sdk/include/reactos/cjkcode.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * PROJECT:     ReactOS header
+ * LICENSE:     LGPL-2.0-or-later (https://spdx.org/licenses/LGPL-2.0-or-later)
+ * PURPOSE:     The encoding, charsets and codepages for Chinese, Japanese and Korean (CJK)
+ * COPYRIGHT:   Copyright 2023 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
```

Comment:
By looking at who wrote those definitions below (that have been moved):
```suggestion
 * COPYRIGHT:   Copyright 2017-2022 Hermès Bélusca-Maïto <hermes.belusca-maito@reactos.org>
                Copyright 2017-2023 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
```